languages/go/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Usage: ./build.sh [--release]
+
+check_command() {
+  if ! command -v "$1" &>/dev/null; then
+    printf '%s\n' "$1 is required to build locally. Please install $2."
+    exit 1
+  fi
+}
+
+check_command cargo Rust
+check_command go Go
+check_command npm Node.js
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+GO_LIB_DIR="$REPO_ROOT/languages/go/internal/cinterface/lib"
+
+pushd "$REPO_ROOT" || exit 1
+printf '%s\n\n' "Cleaning old builds..."
+rm -f languages/go/example/example && rm -rf "$GO_LIB_DIR"
+
+printf '%s\n\n' "Building binaries..."
+
+if [ "$1" = "--release" ]; then
+  cargo build --release
+else
+  cargo build
+fi
+
+npm i && npm run schemas
+
+printf '%s\n\n' "Copying Go bindings to $GO_LIB_DIR..."
+mkdir -p "$GO_LIB_DIR"
+find target/debug -maxdepth 1 -type f -name "libbitwarden_c.*" -exec cp {} "$GO_LIB_DIR"/ \;
```

Comment:
There's a reference to the `target/debug` dir here, which should be `target/release` when building in release mode.

Also, the `libbitwarden_c.*` wildcard copies four files for me (`libbitwarden_c.{a,d,dylib,rlib}`), while only the `dylib` seems to be needed on Mac.

I don't think the extra copies are necessarily a concern, but only `*.dylib` and `*.a` are in the gitignore file, so at the very least we should add the two other types to not clutter the working directory (`*.d` and `*.rlib`).